Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp

```diff
@@ -401,11 +432,11 @@ MediaPlayer::SupportsType MediaPlayerPrivateGStreamerMSE::supportsType(const Med
     if (!ok)
         height = 0;
 
-    if (width > MEDIA_MAX_WIDTH || height > MEDIA_MAX_HEIGHT)
+    if (width > mediaMaxWidth || height > mediaMaxHeight)
         return result;
```

Comment:
If I understand correctly, if VIDEO_DECODING_LIMIT cannot be parsed, IsNotSupported will be always returned. Right?

Wouldn't be better define some "safe" default values?

Reply:
Ups, you are right, will fix that.